README.md

```diff
@@ -0,0 +1,78 @@
+# smartd
+
+This role installs and configures `smartd`. 
+In the default configuration, all devices will be tested and observed, but no mails will be sent.
+
+## Requirements
+
+A Debian or Arch based distribution.
+`/usr/bin/mail` has to exist and be configured to be able to send report mails.
+
+## Role Variables
+
+This role only has one top-level var called `smartd_devices`, which is a dict of dicts to specify device monitoring.
+Each key corresponds to one drive path, while `DEVICESCAN` (the only default entry) may be used to scan for devices.
+After it, all further lines are ignored by smartd, so you probably want to insert it at the end or not at all.
+
+All possible options inside the dict are briefly described here. 
+You can find a detailed wall-of-text for each one on the [smartd man page](https://www.smartmontools.org/browser/trunk/smartmontools/smartd.conf.5.in).
+
+| Name                         |                Mandatory / Default                | Description                                                                                                                                 |
+|:-----------------------------|:-------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------|
+| `type`                       |                      `auto`                       | The type of device, e.g. `ata`, `scsi`, `marvell`, etc.                                                                                     |
+| `nocheck`                    |                  `standby,15,q`                   | Powermode setting. By default, devices in `standby` are only woken up every 15th attempt and skipped check attempts are not logged.         |
+| `check_type`                 |                     `normal`                      | Use `normal` or `permissive` to force SMART checking even if it is not advertised.                                                          |
+| `ata_offline_testing`        |                                                   | Set to `on` or `off` to configure ATA offline testing on smartd startup.                                                                    |
```

Comment:
Isn't there a default?

Reply:
No there isn't. If this option is set, an ATA command will be issued at smartd startup which configures the device to enable or disable offline testing. This setting stays the same through reboots. If this var is not set, the mode on the device won't be changed.